src/integration-tests/modules.js/module.spec.js

```diff
@@ -0,0 +1,36 @@
+const commonHelper = require('@commonTests')
+const schema = require('./responseSchema')
+jest.setTimeout(10000)
+
+describe('Module APIs ', function () {
+	let userDetails
+	beforeAll(async () => {
+		userDetails = await commonHelper.logIn()
```

Comment:
@priyanka-TL  this login function is creating a new user each time. can we change the approach? So that all the the test cases will be checked with a single user itself.